### ISSUE TITLE
Output

### ISSUE THREAD

**Issue by AustinStephen** [mlindauer/AutoFolio#13]
Hello,
I'm trying to use Autofolio for a project on an ASlib scenario. I ran the example ASlib scenario, and information about what appears to be different models gets printed to the console. However, I cannot find a resource that explains how to use this information or what the recommended model should be. Could I get some information about this?
Thank you,
Austin

**Comment by mlindauer** [comment]
Hi Austin,

It's not clear to me what you try to do or what is missing in the readme.
Please provide a minimal code example.

Best,
Marius

PS: Do you do your project with Lars? If yes, I would have guessed that he should be able to help you directly.

**Comment by larskotthoff** [comment]
Yes, and I did have a look at the output, and it also wasn't clear to me. You can probably get all this information from the logs, but for a novice user who doesn't know where to look to get e.g. information on the best configuration it seems that the output can be improved.

Hence the issue -- to make AutoFolio more friendly for novice users.

**Comment by mlindauer** [comment]
Are you looking for a better output or an python API?
I would guess it will be the latter, right? All main functions are actually bundled in https://github.com/mlindauer/AutoFolio/blob/master/autofolio/autofolio.py

If are looking for some special output, I would still recommend to look into this file, since it would also allow you to define your own output if you need some special information.

**Comment by AustinStephen** [comment]
Thank you for the information. The data I was using was just the included examples and titled asp-aslib. I will use the API to try and address where I was confused.
